contracts/protocol/libraries/position/TickPosition.sol

```diff
@@ -78,6 +78,9 @@ library TickPosition {
         unchecked {
             uint64 index = _self.filledIndex;
             uint128 totalSize = 0;
+            if (_self.orderQueue[index].size == _self.orderQueue[index].partialFilled) {
```

Comment:
check if the last partial filled order was canceled or not, if it was then skip it and fill the next order